CHANGELOG.rst

```diff
@@ -16,6 +16,7 @@ Changed
 Fixed
 ~~~~~
 - (`#144 <https://github.com/GranthamImperial/silicone/pull/144>`_) RMS closest no longer causes :class:`pd.core.common.SettingWithCopyWarning` to be raised
+- (`#147 <https://github.com/GranthamImperial/silicone/pull/147>`_) Filter prevents including data from the wrong regions in :class:`DecomposeCollectionTimeDepRatio`.
```

Comment:
```suggestion
- (`#147 <https://github.com/GranthamImperial/silicone/pull/147>`_) Add filter to prevent including data from the wrong regions in :class:`DecomposeCollectionTimeDepRatio`.
```

??

Reply:
I will leave this short as I now have to fit in a description of whatever nonsense is going on with versions affecting a notebook file I didn't originally touch...